src/stream/src/common/table/state_table.rs

```diff
@@ -691,7 +720,27 @@ impl<S: StateStore> StateTable<S> {
                 }
             }
         }
+        if let Some(range_end_suffix) = range_end_suffix {
```

Comment:
Does it means that we must delete-ranges for every epoch ?

Reply:
> Does it means that we must delete-ranges for every epoch ?

fixed